test/integration/go.mod

```diff
@@ -5,45 +5,49 @@ go 1.17
 require (
 	github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test v0.0.0-20220315183512-768ecce22bc3
```

Comment:
Since you are bumping deps, we have started releasing tagged versions for blueprint-test too and would be good to update.